utils/rocvideodecode/roc_video_dec.h

```diff
@@ -76,6 +78,13 @@ typedef enum OutputSurfaceMemoryType_enum {
 #endif
 #define ERR(X) std::cerr << "[ERR] "  << " {" << __func__ <<"} " << " " << X << std::endl;
 
+// timing for session overhead
+#define START_TIMER auto start = std::chrono::high_resolution_clock::now();
```

Comment:
instead of using these definitions (i.e.,  START_TIMER/STOP_TIMER ), add two private functions to the RocVideoDecoder class. 
```
std::chrono::_V2::system_clock::time_point StartTimer();
double StopTimer(const std::chrono::_V2::system_clock::time_point &start_time);
```